filebeat/datadog_checks/filebeat/filebeat.py

```diff
@@ -210,11 +211,16 @@ def _compile_regexes(self):
 
 
 class FilebeatCheck(AgentCheck):
+
+    METRIC_PREFIX = "filebeat."
+
     def __init__(self, *args, **kwargs):
         AgentCheck.__init__(self, *args, **kwargs)
         self.instance_cache = {}
 
     def check(self, instance):
+        normalize_metrics = is_affirmative(instance.get("normalize_metrics"), False)
```

Comment:
I should not make inline suggestions haha, I also had a typo the first time
```suggestion
        normalize_metrics = is_affirmative(instance.get("normalize_metrics", False))
```